recipes-core/images/iot2050-image-example.bb

```diff
@@ -13,7 +13,7 @@ require recipes-core/images/iot2050-package-selections.inc
 
 DESCRIPTION = "IOT2050 Debian Example Image"
 
-DEPENDS += "openssl"
+DEPENDS_class-sdk += "openssl-native"
```

Comment:
Correct syntax would be `DEPENDS:class-sdk`, but in this case you shouldn't need this at all. `openssl` is in `ISAR_REBUILD_ESSENTIAL_PKGS`, and all variants should be built before any other packages (`do_prepare_build` depends on them) or images (`do_rootfs_install` depends on them).

Reply:
Ah, left-over. Will drop.